configuration.md

```diff
@@ -0,0 +1,433 @@
+# IRONdb Configuration Options
+
+IRONdb is implemented using
+[libmtev](https://github.com/circonus-labs/libmtev/), a framework for building
+high-performance C applications. You may wish to review the libmtev
+[configuration documentation](http://circonus-labs.github.io/libmtev/config/)
+for an overview of how libmtev applications are configured generally.
+
+This document deals with options that are specific to IRONdb, but links to
+relevant libmtev documentation where appropriate.
+
+Default values are those that are present in the default configuration produced
+during initial installation.
+
+## irondb.conf
+
+This is the primary configuration file that IRONdb reads at start. It includes
+additional configuration files which are discussed later.
+
+### snowth
+
+```
+<snowth lockfile="/irondb/logs/snowth.lock" text_size_limit="512">
+```
+
+IRONdb's libmtev application name. This is a required node and **must not** be
+changed.
+
+#### snowth lockfile
+
+Path to a file that prevents multiple instances of the application from running
+concurrently. You should not need to change this.
+
+Default: `/irondb/logs/snowth.lock`
+
+#### snowth text_size_limit
+
+The maximum length of a text-type metric value. Text metric values longer than
+this limit will be truncated.
+
+Default: 512
+
+> Text-type metrics are not currently supported in IRONdb, since Graphite has
```

Comment:
"Text-type metrics are supported in IRONdb but graphite currently has no way to render these when using a Storage Finder plugin"